src/app/shared/pipes/order-by.pipe.ts

```diff
@@ -0,0 +1,37 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import Course from '../../pages/courses-page/components/course/course.types';
+
+@Pipe({
+  name: 'orderBy',
+})
+export class OrderByPipe implements PipeTransform {
+
+  transform(array: Array<Course>, order = ''): any[] {
```

Comment:
It's better to make OrderByPipe reusable. It means you need to provide arguments: `transform(array: any[], sortOrder: string, sortBy: string): any[]`

Reply:
done